### ISSUE TITLE
Re-Issue During Validation

### ISSUE THREAD

**Issue by NPPerkins** [lukesUbuntu/gogetsslv2#4]
During the validation process of an SSL certificate you're not able to perform a re-issue. Therefor when the order is being validated the re-issue tab should showcase information that you'll have to wait until the certificate is issued to perform an issue.

**Comment by lukesUbuntu** [comment]
Yes will add this in, as it currently doesn't make sense to be able to try to re-issue. Was going to implement for re-send email as a replacement or for dns/http display pending authorization message

**Comment by lukesUbuntu** [comment]
resolved. will push after some tidy ups as i have started on #3
